base/functors.jl

```diff
@@ -67,6 +67,9 @@ call(::LDivFun, x, y) = x \ y
 immutable IDivFun <: Func{2} end
 call(::IDivFun, x, y) = div(x, y)
 
+immutable DotIDivFun <: Func{2} end
+call(::IDivFun, x, y) = x .÷ y
```

Comment:
Yeah I suspect that's a typo? And a sign that the tests of broadcasting `rem` aren't thorough enough to catch the difference between `rem(x,y)` and `x.%y`.